frontend/src/openapi/v2/services/MetadataService.ts

```diff
@@ -118,7 +118,7 @@ export class MetadataService {
      */
     public static getFileMetadataApiV2FilesFileIdMetadataGet(
         fileId: string,
-        allVersions: boolean = false,
+        allVersions: boolean = true,
```

Comment:
@longshuicy I enabled this here to send the version with the request to update metadata